fencing/commands.c

```diff
@@ -988,43 +988,58 @@ dynamic_list_search_cb(GPid pid, int rc, const char *output, gpointer user_data)
 
 /*!
  * \internal
- * \brief Checks to see if an identical device already exists in the device_list
+ * \brief Returns true if any key in first is not is second or second has a different value for key
```

Comment:
Yeah it should probably be: Returns true if any key in first is not in second or second has a different value for a key